Tests/scripts/utils/log_util.py

```diff
@@ -187,8 +189,8 @@ def _add_logger(self, thread_name: str):
         log_queue = queue.Queue(-1)
         queue_handler = QueueHandler(log_queue)
         queue_handler.setLevel(logging.DEBUG)
-
         logger = logging.getLogger(thread_name)
+        logger.propagate = False
```

Comment:
why is this change?

Reply:
Because without it the log message is passed to the root logger as well which logs it to stdout too, and we get duplicated logs each time.